pandas/tests/groupby/test_transform.py

```diff
@@ -317,6 +317,19 @@ def test_dispatch_transform(tsframe):
     tm.assert_frame_equal(filled, expected)
 
 
+def test_transform_fillna():
```

Comment:
yeah need to test for ffill, bfill, shift as well i think

Reply:
added those tests